docs/docs/references/connectors/README.mdx

```diff
@@ -0,0 +1,106 @@
+import ConnectorPlatformIntro from './fragments/_connector_platform_intro.mdx';
+import ConnectorStorageIntro from './fragments/_connector_storage_intro.mdx';
+import ConnectorTypeIntro from './fragments/_connector_type_intro.mdx';
+
+# 🔌 Connectors
+
+## Introduction
+
+### What is a connector?
+
+_Connectors_ play a critical role in Logto. With their help, Logto enables end-users to use SMS-based or email-based passwordless registration and the capabilities of signing in with social accounts. With the increasing popularity of websites and applications, passwordless and social sign-ins allow users to avoid managing numerous accounts and passwords.
+
+Sounds fascinating :hushed:, Uh? See [Configure connectors](../../recipes/configure-connectors/) if you want to set up an existing connector. If you can not find the connector you want to set up, you may develop those connectors by following the guides in [Create your connector](../../recipes/create-your-connector/).
+
+## Compositions
+
+_Connectors_ are stored both locally and remotely:
+
+<ConnectorStorageIntro />
+
+## Connector's local storage: _ConnectorMetadata_
+
+### id
+
+_**id**_ is an _unique_ string-typed key to identify connector in Logto.
+
+It's assigned by the developers of each connector and will be uploaded to DB.
+
+### target
+
+_**target**_ is a lowercase string to distinguish the social identities source of the social connector.
+For email and SMS connectors, _**target**s_ work more like a "human readable id".
+
+For example, both _WeChat web connector_ and _WeChat native connector_'s _**target**_ are `wechat`. _Aliyun direct mail connector_'s _**target**_ is `aliyun-dm` and _Aliyun SMS connector_'s _**target**_ is `aliyun-sms`.
+
+### type
+
+_**type**_ should be one of the following connector types:
+
+<ConnectorTypeIntro />
+
+### platform
+
+_**platform**_ should be one of the following values:
+
+<ConnectorPlatformIntro />
+
+### name
+
+_**name**_ is a Javascript/Typescript object whose keys are i18n country codes and values are connectors' display name.
+
+### logo
+
+_**logo**_ is an URL or relative path of connector's logo (light mode by default).
+
+### logoDark
+
+_**logo**_ is an _nullable_ URL or relative path of connector's dark mode logo.
+
+:::note
+Usually, _**logo**s_ always exist, and _**logoDark**s_ only exist for some exceptional cases when the light mode logos may become obscure in dark mode.
```

Comment:
I think just mentioning logo is required, logoDark is optional should be good enough. We display logo in both light and dark mode if no logoDark is provided.  

`some exceptional cases when the light mode logos may become obscure in dark mode` s` is objective. All we need to show is we support showing different logos under the different color themes.  Leave all the judgment to the users themselves.

Reply:
Good call, updated.